crates/nu-cli/src/completions/operator_completions.rs

```diff
@@ -40,57 +40,57 @@ impl Completer for OperatorCompletion {
+                ("==", "Equal to"),
+                ("!=", "Not equal to"),
+                ("//", "Floor division"),
+                ("<", "Less than"),
+                (">", "Greater than"),
+                ("<=", "Less than or equal to"),
+                (">=", "Greater than or equal to"),
+                ("mod", "Floor division remainder (Modulo)"),
+                ("**", "Power of"),
+                ("bit-or", "Bitwise OR"),
+                ("bit-xor", "Bitwise exclusive OR"),
+                ("bit-and", "Bitwise AND"),
+                ("bit-shl", "Bitwise shift left"),
+                ("bit-shr", "Bitwise shift right"),
             ],
             Expr::String(_) => vec![
-                ("=~", "Regex Match / Contains"),
-                ("!~", "Not Regex Match / Not Contains"),
-                ("in", "In / Contains (doesn't use regex)"),
+                ("=~", "Contains regex match"),
+                ("!~", "Does not contain regex match"),
+                ("in", "In / Contained by (no regex)"),
                 (
                     "++",
                     "Appends two lists, a list and a value, two strings, or two binary values",
                 ),
-                ("not-in", "Not In / Not Contains (doesn't use regex"),
-                ("starts-with", "Starts With"),
-                ("ends-with", "Ends With"),
+                ("not-in", "Not in / Not contained by (no regex)"),
+                ("starts-with", "Starts with"),
+                ("ends-with", "Ends with"),
             ],
             Expr::Float(_) => vec![
-                ("+", "Plus / Addition"),
-                ("-", "Minus / Subtraction"),
+                ("+", "Add (Plus)"),
+                ("-", "Subtract (Minus)"),
                 ("*", "Multiply"),
                 ("/", "Divide"),
-                ("==", "Equal"),
-                ("!=", "Not Equal"),
-                ("//", "Floor Division"),
-                ("<", "Less Than"),
-                (">", "Greater Than"),
-                ("<=", "Less Than or Equal to"),
-                (">=", "Greater Than or Equal to"),
-                ("mod", "Modulo"),
-                ("**", "Pow"),
+                ("==", "Equal to"),
+                ("!=", "Not equal to"),
+                ("//", "Floor division"),
+                ("<", "Less than"),
+                (">", "Greater than"),
+                ("<=", "Less than or equal to"),
+                (">=", "Greater than or equal To"),
+                ("mod", "Floor division remainder (Modulo)"),
+                ("**", "Power of"),
             ],
             Expr::Bool(_) => vec![
-                ("and", "Checks if both values are true."),
-                ("or", "Checks if either value is true."),
-                ("xor", "Checks if one value is true and the other is false."),
-                ("not", "Negates a value or expression."),
+                ("and", "Both values are true (w/short-circuit)"),
```

Comment:
I'm not a fan of this "w/" abbreviation. It assumes people understand it which I don't think is a safe assumption.

Reply:
Fair enough - Perhaps not as universal as I assume.  You are okay with it fully expanded to *"(with short-circuit)"*, then right?

Reply:
It's probably ok for now, we can always tweak it later, but I'm not sure "with short-circuit" indicates that a short-circuit is needed somehow or using this will automatically short-circuit by default.

Reply:
Agreed - It's ambiguous.  Is *"(short-circuits)"* any better?

Reply:
sure, we can run with that and see how it goes.

Reply:
I went a bit longer to be more clear. It's still shorter than the `++` operator description, so it's not too long.

```
and     Both values are true (short-circuits when first value is false)
or      Either value is true (short-circuits when first value is true)
```